templates/repo/file_info.tmpl

```diff
@@ -30,6 +30,16 @@
 			{{ctx.Locale.Tr "repo.executable_file"}}
 		</div>
 	{{end}}
+	{{if .IsVendored}}
+		<div class="file-info-entry">
+			{{ctx.Locale.Tr "repo.vendored"}}
```

Comment:
`repo.vendored` doesn't seem a good key

Actually, you could reuse the `repo.diff.vendored` and reuse that UI.

Reply:
These strings are lowercase. Should I make it uppercase to match the rest of the Header?

Reply:
> reuse that UI.

What do you think about this? Reuse that UI and reuse those texts.

Reply:
The UI looks a little bit different than the UI on the view file page.